master/internal/resourcemanagers/agent/agent.go

```diff
@@ -69,6 +70,8 @@ type (
 		opts *aproto.MasterSetAgentOptions
 
 		agentState *AgentState
+
+		db db.DB
```

Comment:
per the previous comment - let's not add `db` here.

Reply:
done